spec/factories/states.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :state do
+    name { "MyString" }
+    code { "MyString" }
+  end
```

Comment:
use sequence or faker here to create different states

Reply:
done